modules/ROOT/pages/using-api-alerts.adoc

```diff
@@ -2,46 +2,42 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: alerts,api, api alerts
+:keywords: alerts, api, api alerts
 
-An API alert is an alarm that flags one of the following problems:
+Anypoint Platform alerts you about request behavior, performance issues, actions related to client application contracts, and runtime events. The platform supports three types of alerts: API alerts, contracts alerts, and runtime alerts. API alerts and contracts alerts are managed by Anypoint API Manager (API Manager). Runtime alerts are managed by Anypoint Runtime Manager (Runtime Manager).
 
-* The API request violates a policy.
-* Requests received by the API exceed a given number within a period of time.
-* The API returns a specified HTTP error code.
-* The API response time exceeds a certain amount.
+== API Alerts
+ 
+Add and configure API alerts that flag unusual API behavior. Examples include:
```

Comment:
I am making this suggestion only because I "tripped" over "add and configure" leading to "examples" (rather than "behavior" leading to "examples).
But I realize that this just might be Morning Brain. Your choice!
```suggestion
Add and configure API alerts that flag unusual API behavior, such as:
```

Reply:
Fixed